tools/experiments/experiments.js

```diff
@@ -138,6 +143,34 @@ function build() {
   } else {
     redirect.setAttribute('hidden', '');
   }
+
+  const rtvInput = document.getElementById('rtv');
+  const rtvButton = document.getElementById('rtv-submit');
+  rtvInput.addEventListener('input', () => {
+    rtvButton.disabled = !(
+      rtvInput.value == '' || RTV_PATTERN.test(rtvInput.value)
+    );
+    rtvButton.textContent = rtvInput.value == '' ? 'opt-out' : 'opt-in';
+  });
+  rtvButton.addEventListener('click', () => {
+    if (rtvInput.value == '') {
```

Comment:
Does the field default to the currently opted-in RTV value? My concern is that if it's initially empty and someone hits enter, would it opt them out unintentionally?

Reply:
Also nit: `if (!rtvInput)`

Reply:
> Does the field default to the currently opted-in RTV value?

Yes

> My concern is that if it's initially empty and someone hits enter, would it opt them out unintentionally?

There is a confirmation popup